libqtile/widget/accessDots.py

```diff
@@ -0,0 +1,56 @@
+# Copyright (c) 2021, Bhavuk Sharma
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+# modification suggested by elParaguayo (https://github.com/elParaguayo)
+# This feature is also included in iOS 14 and Android 12 and inspired from an app AccessDots for Android.
+
+from libqtile.widget import base
+import os
+
+
+class active(base.ThreadPoolText):
```

Comment:
Needs a better name and should be capitalised. Personally, I don't like AccessDots as that's just copying the Android app name and it gives the user no idea what it does.

Reply:
I would also add a warning that this will only tell you the status of the device when the widget polls the device (default once a second). It does not guarantee that your device is not being accessed in between updates.

Reply:
Can u please suggest a good name. I am very bad when it comes to naming.

Reply:
**IOaccess** sounds good!